source/database-connections/save-relational-connection.txt

```diff
@@ -0,0 +1,85 @@
+
+.. _rm-save-relational-connection:
+
+============================================
+Save a Relational Database Connection
+============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can save a new relational database connection from the Relational Migrator
+:guilabel:`Connections` page, or when creating or modifying a data migration job.
```

Comment:
There are two other options:
1. When creating a project using a live database connection (I think this will be the most popular option)
2. From the Connections bar.